utils/python/CIME/utils.py

```diff
@@ -508,6 +508,7 @@ def get_timestamp(timestamp_format="%Y%m%d_%H%M%S", utc_time=False):
 def get_project(machobj=None):
     """
     Hierarchy for choosing PROJECT:
+    0. Command line flag to create_newcase or create_test
```

Comment:
It's not relevant to this commit, but you aren't going to penalize me for improving documentation are you?